tests/Skills/Child/MainDialog.cs

```diff
@@ -38,6 +38,11 @@ private async Task<DialogTurnResult> SignInStepAsync(WaterfallStepContext contex
 
         private async Task<DialogTurnResult> ShowTokenResponseAsync(WaterfallStepContext context, CancellationToken cancellationToken)
         {
+            if (context.Context.Activity.DeliveryMode == DeliveryModes.ExpectReplies)
```

Comment:
A comment here would be helpful to explain what happens if you don't set the DeliveryMode to `null`.